rhel8/profiles/cis.profile

```diff
@@ -601,87 +601,88 @@ selections:
 
     ### 4.1.9 Ensure discretionary access control permission modification
     ###       events are collected (Scored)
-
+    # NEEDS RULE - https://github.com/ComplianceAsCode/content/issues/5509
     
     ### 4.1.10 Ensure unsuccessful unauthorized file access attempts are
     ###        collected (Scored)
-
+    # NEEDS RULE - https://github.com/ComplianceAsCode/content/issues/5510
 
     ### 4.1.11 Ensure events that modify user/group information are
     ###        collected (Scored)
-
+    # NEEDS RULE - https://github.com/ComplianceAsCode/content/issues/5511
```

Comment:
We have rules for this, for example audit_rules_etc_group_openat etc

Reply:
Perhaps! Feel free to self assign that ticket and review/patch as applicable.